mwdb/model/user.py

```diff
@@ -43,6 +50,7 @@ class User(db.Model):
     logged_on = db.Column(db.DateTime)
     set_password_on = db.Column(db.DateTime)
 
+    favorites = db.relationship('Object', secondary=favorites, backref='users', lazy='selectin')
```

Comment:
`Object.users` should renamed to something more related with 'favorites' e.g. `followers` (?)